lib/content-services/src/lib/category/categories-management/categories-management.component.ts

```diff
@@ -177,10 +177,18 @@ export class CategoriesManagementComponent implements OnInit, OnDestroy {
         this.cancelExistingCategoriesLoading$.complete();
     }
 
+    get isNameCategoryVisible(): boolean {
```

Comment:
missing documentation comments for public api

Reply:
I have updated documentation here https://github.com/Alfresco/alfresco-ng2-components/blob/62bdd33c10964bcb631cfe050e39c5190439926e/docs/content-services/components/categories-management.component.md
Please review it

Reply:
it needs to be the code comment above each public api, we generate docs later